### PR TITLE
suggested changes to readme and class_9_intro_cv_cnns.ipynb

### PR DESCRIPTION
**Pull request by jeremy-large**
small edits that helped me to get this going on my local PC

### CODE REVIEW

File: notebooks/class_9_intro_cv_cnns.ipynb [hunk 1]
```diff
@@ -525,16 +500,32 @@
    "outputs": [],
    "source": [
     "###If using Binder:\n",
-    "os.makedirs(os.path.join(root,'data','mnist','3.0.1'))\n",
+    "datadir = os.path.join(root,'data','mnist','3.0.1')\n",
```
Comment (jeremy-large) [inline notebooks/class_9_intro_cv_cnns.ipynb:503]: These are the only material changes in the notebook. The rest seems to be stripping an 'id' field from each cell which I could not avert but seems to be innocuous

**Comment by jeremy-large** [comment]
Incidentally, this resolves #4

**Comment by jeremy-large** [comment]
Looks like I need to populate, locally,

`ox-sbs-ml-bd/data/flower_photos/**/*.pdf`

OK - so that involves a tar ball or such. I'm not sure how to handle this on the PC

**Comment by Lkruitwagen** [comment]
Hey @jeremy-large, so do I understand correctly that you're not able to run this cell on PC?

`!wget -c https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz -O - | tar -xz -C {root}/data` 

If so I can try to find a windows work-around. Will try running the whole nb on my windows partition.

File: notebooks/class_9_intro_cv_cnns.ipynb (hunk 1 above)
Comment (Lkruitwagen) [reply notebooks/class_9_intro_cv_cnns.ipynb:503]: great, thanks for this. Any idea how I can save a notebook without the "id" field?

File: README.md [hunk 2]
```diff
@@ -4,18 +4,19 @@
...
 
 Here's a [conda](https://www.anaconda.com/products/individual) command to create a suitable environment (called `ml_bd`) for this code locally:
-* `conda create -n ml_bd -c anaconda python=3.6 scikit-learn=0.22.1 statsmodels xlrd=1.1.0 pandas matplotlib jupyterlab nbconvert`
+* `conda create -n ml_bd -c anaconda python scikit-learn statsmodels xlrd pandas matplotlib jupyterlab nbconvert`
 
 
 Further commands that create another conda environment (`tf_ml_bd`) with tensorflow & keras added (to go beyond Class 7):
 * `conda create --clone ml_bd --name tf_ml_bd`
-* `conda install -c conda-forge tensorflow python-graphviz tensorflow-datasets -n tf_ml_bd scikit-image`
+* `conda install -c conda-forge tensorflow python-graphviz tensorflow-datasets scikit-image -n tf_ml_bd`
```
Comment (Lkruitwagen) [inline README.md:12]: Ah, good catch, thank you.

**Review by Lkruitwagen — APPROVED** [review]

**Pull request merged**